### ISSUE TITLE
Create simple server

### ISSUE THREAD

**Issue by yangli-io** [yangli1990/Node-Server#1]
@KarthikBangaloreVenkatarama Create a simple node server with node's http.

Start by using npm init

**Comment by yangli-io** [comment]
Create a new branch and when you're done set up a pull request before we merge into the master branch

**Comment by yangli-io** [comment]
Great work, don't merge until you've gotten some review
